shared/js/l10n.js

```diff
@@ -2357,7 +2324,7 @@
     }
 
     if (typeof value === 'string') {
-      if (!entity.overlay) {
+      if (value.indexOf('<') === -1 && value.indexOf('&') === -1) {
```

Comment:
Did you mean `Warsaw <3 San Francisco`? :)

I think we should support things like `< span >` which is valid HTML.